### PR TITLE
Adding UI for extra cards

### LINKED ISSUE #8: Option to add image card at specific positions
This would be a solution to be able to add ads at next to cards instead of using a banner on top of the page. We begin with a simplified solution and places ads as the first card (place 1) and on place 12. This is similar to the solution used on category pages today.

We think we can expect higher click rate. But it will also free some space on desktop because the banner takes most of the screen on laptops, and for everyone it might not be very effective because you quickly focus on the products below it.

The image will have a size of 230 x 460 px and full height will always be visible. The left and right side will be cropped. This size is calculated based on a minimum of 1:2 aspect ratio and max of 2:3. FYI, this results into some images being cropped down in normal cases. In some cases product cards (next to the ad) will have unexpected amount of extra information and the width will be even smaller.

From a dev perspective I think it will be enough to use:
```
background-size: cover;
background-repeat: no-repeat;
background-position: center top;
```

A "Shop now" (Handla nu) button is automatically added to the bottom of the card in the same position as action buttons on product cards. This button+label is same for all ads.

Data needed:
- img url
- alt/title (can't be used on background images...)
- click url

### LINKED ISSUE #9: Information card with other data
This card need:
- img url
- img alt text
- header
- text (with support for paragraph breaks)
- link url
- button text
- position (optional) (1 or 12 as mentioned in #8)
- label (optional)
- brand (optional)

The design will be simple and flexible and the button will be placed aligned with buttons on other cards.

### PR DESCRIPTION
**Pull request by coopjakob**
Requirements:
* Ad cards, fixes #8 
* Info cards, fixes #9

### CODE REVIEW

**Comment by benjick** [comment]
There's a few semicolons in the sass-code

**Comment by coopjakob** [comment]
> There's a few semicolons in the sass-code

Fixed?

File: components/InfoCard.vue [hunk 1]
```diff
@@ -17,3 +49,40 @@ export default Vue.extend({
...
+    padding: 15px
+
+figure
+  margin: 0 //reset
+  position: relative
+  width: 100%
+  padding-top: 100%
+
+  img
+    position: absolute;
```
Comment (benjick) [inline components/InfoCard.vue]:
```suggestion
    position: absolute
```

File: components/InfoCard.vue [hunk 2]
```diff
@@ -17,3 +49,40 @@ export default Vue.extend({
...
+    max-width: 80%
+    max-height: 150px
+    bottom: 0;
```
Comment (benjick) [inline components/InfoCard.vue]:
```suggestion
    bottom: 0
```

**Pull request merged**